src/commands/init.ts

```
@@ -47,7 +59,7 @@
 
 	const { projectType, taskRunners } = await getUserInput();
 
-	await runInitializationTasks(projectType, taskRunners);
+	await runInitializationTasks(projectType, taskRunners, commandOptions);
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `"rbxts" | "luau"`.

(ts/no-unsafe-argument)